.chloggen/mongodbatlasreceiver-checkapi-26304.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: mongodbatlasreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Do not export the functions `GetTestAuditEvent4_2`, `GetTestAuditEvent5_0`, `GetTestEvent4_2`, `GetTestEvent4_4` and rename struct to pass checkapi.
```

Comment:
```suggestion
note: Rename struct to pass checkapi.
```

Reply:
`GetTestAuditEvent4_2`, `GetTestAuditEvent5_0`, `GetTestEvent4_2`, `GetTestEvent4_4` are used in tests only so probably no need to mention them.